servicetalk-grpc-gradle-plugin/src/main/groovy/io/servicetalk/grpc/gradle/plugin/ServiceTalkGrpcPlugin.groovy

```diff
@@ -54,7 +54,7 @@ class ServiceTalkGrpcPlugin implements Plugin<Project> {
       def serviceTalkGrpcProtoc = "servicetalk-grpc-protoc"
       def serviceTalkVersion = pluginProperties."implementation-version"
       def serviceTalkProtocPluginPath = extension.serviceTalkProtocPluginPath
-      if (!serviceTalkVersion && !serviceTalkProtocPluginPath) {
+      if (!isVersion(serviceTalkVersion) && !serviceTalkProtocPluginPath) {
```

Comment:
`serviceTalkVersion` is always not empty. But it could have `${projectVersion}` as a value instead of the actual version (e.g. `0.26.0-SNAPSHOT`) when this code executes before `processResources`.